src/app/pages/news/11.2/04-engine-cockpit.md

```diff
@@ -1,6 +1,13 @@
 ## Engine Cockpit {#engineCockpit}
 
-Blah Engine Cockpit blah
+We added awesome new world-class monitoring, user management, operations, and configuration features to the Engine Cockpit.
+
+- **Configuration**: Use the new **System/SSL** view to manage your certificates. The new **System/Config File Editor** helps you view and edit any configuration file of Axon Ivy. Press CTRL-SPACE to get content assistance. 
+- **User Management**: The user details view now shows you the Substitutes and Absences of a user. The security identity provider configuration view has been reimplemented with a new LDAP browser and Azure AD browser.
```

Comment:
I'd emphasize the new feature Azure AD browser like this: (ldap is basically still the same as before)

... has been re-implemented and is now featuring a directory browser for Azure AD.